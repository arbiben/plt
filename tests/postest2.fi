$ tests indexing $
int x?

char func(){
new_array[0]? $this works$
$int x = thing.take? does not work (regular assignment does work but not with a typ before it$
x = new_array[1]?
}
