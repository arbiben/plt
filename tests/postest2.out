
int x?

char func()
{
new_array[0]?
int x = thing.take?
x = new_array[1]?
char y = 'a'?
if (x is not y)
print(T)?
}

